### ISSUE TITLE
Fine-tuning with custom dataset

### ISSUE THREAD

**Issue by chophilip21** [Media-Smart/vedastr#73]
Hello, I'm trying to test Vedastr for reading ID tags on animals (mostly cows).

Would it be possible to train on custom dataset? If so, what are the procedures?

I would like to use your pretrained weights to fine-tune on my custom dataset, and I am not sure where to look into exactly.

**Comment by hungduong-vn** [comment]
Yeah, I am in need of a scene text recognition module but fit on a Vietnamese dataset. Therefore, being able to fine tune the vedastr's models to different datasets would be nice since they are already trained on diverse and standardized datasets for scene text recognition task.

@ChaseMonsterAway Can you please provide us the steps to perform fine-tuning of your pretrained models?
